io.catenax.material_demand/1.0.0/MaterialDemand.ttl

```diff
@@ -0,0 +1,376 @@
+   samm:characteristic :BPNLTrait ;
+   samm:exampleValue "BPNL6666666666YY" .
+
+:unitOfMeasure a samm:Property ;
+   samm:preferredName "Unit of Measure"@en ;
+   samm:description "Unit of measurement for demand quantities."@en ;
+   samm:characteristic ext-quantity:ItemUnitEnumeration ;
+   samm:exampleValue "unit:piece"^^samm:curie .
+
+:materialNumberCustomer a samm:Property ;
+   samm:preferredName "Customer Material Number"@en ;
+   samm:description "Material identifier as assigned by customer. This material number identifies the material (as planned) in customer's database. Must be unique for each Material Demand in the customer-supplier relationship."@en ;
+   samm:characteristic :MaterialNumber ;
+   samm:exampleValue "MNR-7307-AU340474.002" .
+
+:materialNumberSupplier a samm:Property ;
+   samm:preferredName "Supplier Material Number"@en ;
+   samm:description "Material identifier as assigned by supplier. This material number identifies the material (as planned) in supplier's database."@en ;
+   samm:characteristic :MaterialNumber ;
+   samm:exampleValue "MNR-8101-ID146955.001" .
+
+:materialDescriptionCustomer a samm:Property ;
+   samm:preferredName "Customer Material Description"@en ;
+   samm:description "Description of the material."@en ;
+   samm:characteristic :MaterialDescription ;
+   samm:exampleValue "Spark Plug" .
+
+:changedAt a samm:Property ;
+   samm:preferredName "Changed At"@en ;
+   samm:description "Point in time when the content (any property according to the data model) of the material demand was changed, at the customer, either by a human user or an automated process."@en ;
+   samm:characteristic :Timestamp ;
+   samm:exampleValue "2023-11-05T08:15:30.123-05:00"^^xsd:dateTimeStamp .
+
+:materialGlobalAssetId a samm:Property ;
+   samm:preferredName "Material Number UUID of customers BOM as planned twin used in Catena-X"@en ;
+   samm:description "Material identifier used uniquely within Catena-X to identify the Material (e.g. PartAsPlanned)."@en ;
+   samm:characteristic :MaterialGlobalAssetIdTrait ;
+   samm:exampleValue "urn:uuid:48878d48-6f1d-47f5-8ded-a441d0d879df" .
+
+:demandRate a samm:Property ;
+   samm:preferredName "Demand Rate"@en ;
+   samm:description "Describes the demand rate or granularity of the demand information. It defines how to interpret the point in time."@en ;
+   samm:characteristic :DemandRateEnumeration .
+
+:unitOfMeasureIsOmitted a samm:Property ;
+   samm:preferredName "Unit Of Measure Is Omitted"@en ;
+   samm:description "Explicit indicator of wether the unit of measure is left out of the payload intentionally. If \"true\" it means the sending application sends demand values without unit of measure intentionally and the unit of measure MUST NOT be contained in the payload. If \"false\" a unit of measure MUST be supplied."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue false .
```

Comment:
Why is this property necessary, when `unitOfMeasure` is already optional and allows to express exactly the same as this property? All "features" `unitOfMeasureIsOmitted` adds is an extra validation on the API level. Otherwise, it offers no benefit. If `unitOfMeasure` is missing in the payload, it obviously was omitted, no need for `unitOfMeasureIsOmitted`. If `unitOfMeasure` _is_ present in the payload, well, then it obviously was not omitted, and the extra property is also not necessary. I really don't see why we need this redundant property...

Reply:
We'll keep `unitOfMeasureIsOmitted` as discussed in Architecture Circle today